diagnostic.md

```diff
@@ -150,7 +178,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+they will be inline starting frim the left
+it a box(div) wont fit in the line the line it will start a new line under it.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).